pocs/linux/kernelctf/CVE-2023-4569_lts/docs/exploit.md

```diff
@@ -0,0 +1,451 @@
+# Exploit detail about CVE-2023-4569
+If you want to get some base information about CVE-2023-4569, please read [vulnerability.md](./vulnerability.md) first.
+
+## Background
+nftables is a netfilter project that aims to replace the existing {ip,ip6,arp,eb}tables framework, providing a new packet filtering framework for {ip,ip6}tables, a new userspace utility (nft) and A compatibility layer. It uses existing hooks, link tracking system, user space queuing component and netfilter logging subsystem.
+
+It consists of three main components: kernel implementation, libnl netlink communication and nftables user space front-end. The kernel provides a netlink configuration interface and runtime rule set evaluation. libnl contains basic functions for communicating with the kernel. The nftables front end is for user interaction through nft.
+
+nftables implements data packet filtering by using some components like `table`, `set`, `chain`, `rule`.
+
+## Cause anaylysis
+
+In function nft_map_catchall_deactivate, it checks if an elem in set->catchall_list can be deactivated by this:
+
+```
+if (!nft_set_elem_active(ext, genmask))
+    continue;
+```
+but it should check like code in function  nft_set_catchall_flush:

```

Comment:
Could you please synchronise this with update on fix done in vulnerability.md file?